content/en/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -231,6 +231,47 @@ Similar to the container logs, you should rotate system component logs in the `/
 In Kubernetes clusters created by the `kube-up.sh` script, log rotation is configured by the `logrotate` tool.
 The `logrotate` tool rotates logs daily, or once the log size is greater than 100MB.
 
+## Log query
+
+{{< feature-state for_k8s_version="v1.27" state="alpha" >}}
+
+To help with debugging issues on nodes, Kubernetes v1.27 introduces a feature that allows viewing logs of services
+running on the node. To use the feature, ensure that the `NodeLogQuery`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) kubelet options, `EnableSystemLogHandler`
+and `EnableSystemLogQuery`, are enabled. The feature is available only to cluster administrators.
+
+You can now query services on the node:
+```shell
+kubectl get --raw "/api/v1/nodes/<insert-node-name-here>/proxy/logs/?query=<insert-service-name-here>"
```

Comment:
```suggestion
# Fetch kubelet logs from a node named node-1.example
kubectl get --raw "/api/v1/nodes/node-1.example/proxy/logs/?query=kubelet"
```

I think the kubelet is a good example of a service we can expect to find on nodes.